build.sbt

```diff
@@ -131,7 +131,8 @@ lazy val D = new {
 
   // Customized version of numbus jose jwt
   // from https://github.com/goncalo-frade-iohk/Nimbus-JWT_Fork/commit/8a6665c25979e771afae29ce8c965c8b0312fefb
-  val nimbusJwt: ModuleID = "io.iohk.atala" % "nimbus-jose-jwt" % V.nimbusJwt
+  // val nimbusJwt: ModuleID = "io.iohk.atala" % "nimbus-jose-jwt" % V.nimbusJwt
+  val nimbusJwt: ModuleID = "com.nimbusds" % "nimbus-jose-jwt" % V.nimbusJwt
```

Comment:
oh.. 
I see, we switched back to the official library
well done!

Reply:
This is came out of the discussion on the slack thread since when we move to the new| https://github.com/eu-digital-identity-wallet/eudi-lib-jvm-sdjwt-kt/blob/main/gradle/libs.versions.toml#L8  they use official version and we wont able to use both libraries @yshyn-iohk